db/upgrade.php

```diff
@@ -72,6 +72,18 @@ function xmldb_auth_saml2_upgrade($oldversion) {
         upgrade_plugin_savepoint(true, 2016031701, 'auth', 'saml2');
     }
 
+    if ($oldversion < 2016071201) {
+        // Update plugin configuration settings from auth_saml2 to auth/saml2
+        $rs = $DB->get_recordset_select('config_plugins', 'plugin = ?', array('auth_saml2'));
```

Comment:
There is a bug with this with the 'version', because core stores it as:

59│ 1272 ║ auth_saml2 │ version                             │ 2016071200

We need to leave it alone